whist/server/web_socket/subscriber.py

```diff
@@ -23,5 +23,5 @@ def send(self, event: Event) -> None:
         :param event: Any type of event.
         :return: None
         """
-        wrapped_event = {'name': event.name, 'event': event}
+        wrapped_event = {'name': event.name, 'event': event.json()}
```

Comment:
why are you converting the event into a json string?

Reply:
The default json dumps couldn't handle it internally.

Reply:
Use .dict instead?